docs/query_construction.md

```diff
@@ -0,0 +1,239 @@
+```python
+from rcsbapi.data import Query
+query = Query(
+    input_ids={"entry_id": "4HHB"},
+    input_type="entry",
+    return_data_list=["exptl.method"]
+)
+result_dict = query.exec()
+print(result_dict)
+```
+```json
+{
+  "data": {
+    "entry": {
+      "exptl": [
+        {
+          "method": "X-RAY DIFFRACTION"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Helpful Methods
+There are several methods included to make working with query objects easier. These methods can help you refine your queries to request exactly and only what you want and further understand the GraphQL syntax.
+
+### get_editor_link()
+This method returns the link to a [GraphiQL](https://data.rcsb.org/graphql/index.html) window with the query. From the window, you can use the user interface to explore other fields and refine your query. Method of Query class.
+
+```python
+from rcsbapi.data import Query
+query = Query(
+    input_ids={"entry_id": "4HHB"},
+    input_type="entry",
+    return_data_list=["exptl"]
+)
+editor_link = query.get_editor_link()
+print(editor_link)
+```
+
+### get_unique_fields() <!--Should this be moved outside the schema method?-->
+Given a redundant field, this method returns a list of matching fields in dot notation. You can look through the list to identify your intended field. Method of Schema class.
+
+```python
+from rcsbapi.data import Schema
+schema = Schema()
+schema.get_unique_fields("id")
+```
+
+### find_field_names()
+Given a string, this method will return all fields containing that string, along with a description of each field.
+
+```python
+from rcsbapi.data import Schema
+schema = Schema()
+schema.find_field_names("exptl")
+```
+
+### get_input_id_dict()
+Given an input_type, returns a dictionary with the corresponding keys and descriptions of each key. Method of Schema class.
+
+```python
+from rcsbapi.data import Schema
+schema = Schema()
+schema.get_input_id_dict("polymer_entity_instance")
+```
+
+## Troubleshooting
+### ValueError: Not a unique field
+Some fields are redundant within our GraphQL Data API schema. For example, "id" appears over 50 times. To allow for specific querying, redundant fields are identified by the syntax `<type>.<field name>`. If you request a redundant field without this syntax, a `ValueError` will be returned stating that the field exists, but is redundant. You can then use `get_unique_fields("<field name>")` to find notation that would specify a unique field for the given name.
+
+```python
+from rcsbapi.data import Query
+
+# querying a redundant field
+query = Query(
+    input_ids={"entry_id": "4HHB"},
+    input_type="entry",
+    return_data_list=["id"]
+)
+result_dict = query.exec()
+print(result_dict)
+```
+> \> ValueError: "id" exists, but is not a unique field, must specify further.
+To find valid fields with this name, run: get_unique_fields("id")
```

Comment:
Instead of using ">" at the beginning of the line, can you just surround this with the three tick marks? I.e.,:

\`\`\`
\> ValueError: "id" exists, but is not a unique field, must specify further.
To find valid fields with this name, run: Schema().get_unique_fields("id")
\`\`\`

(Also just a reminder to update the text if you decide to accept my other suggestion for printing this message above)